onnxmltools/convert/keras/operator_converters/SeparableConv.py

```diff
@@ -0,0 +1,144 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import numpy
+import keras
+from distutils.version import StrictVersion
+if StrictVersion(keras.__version__) >= StrictVersion('2.1.5'):
+    from keras.layers import SeparableConv1D, SeparableConv2D
+
+from ....proto import onnx_proto
+from ...common._apply_operation import apply_identity, apply_transpose
+from ...common._registration import register_converter
+from .Dense import _activation_map
+
+
+def convert_keras_separable_conv_core(scope, operator, container, is_transpose, n_dims, input_perm_axes,
+                            output_perm_axes, weight_perm_axes):
+    op = operator.raw_operator
```

Comment:
a lot of duplicates from Conv.py.convert_keras_conv_core. it will be good to merge them

Reply:
resolved?